python/az/aro/azext_aro/_params.py

```diff
@@ -68,6 +68,8 @@ def load_arguments(self, _):
         c.argument('outbound_type',
                    help='Outbound type of cluster. Must be "Loadbalancer" (default) or "UserDefinedRouting".',
                    validator=validate_outbound_type)
+        c.argument('enable_preconfigured_nsg',
```

Comment:
can you make this `arg_type=get_three_state_flag()`? this will align with other true/false/none flags we have.